Example Home Assistant Configuration.yaml

```diff
@@ -27,32 +35,31 @@ light:
       - rainbow  
       - rainbow with glitter  
       - ripple  
-      - sinelon  
-      - solid  
-      - twinkle  
+      - sinelon    
+      - twinkle
+      - sinelon
+      - sine hue
+      - full hue
+      - breathe
+      - hue breathe
+      - Christmas
+      - christmas alternate
+      - random stars
+      - St Patty
+      - Valentine
+      - Turkey Day
+      - Thanksgiving
+      - USA
+      - Independence
+      - Halloween
+      - Go Lions
+      - Hail
+      - Touchdown
+      - Punkin
+      - Lovey Day
+      - Holly Jolly      
     brightness: true
     flash: true
     rgb: true
     optimistic: false
-    qos: 0
-
-input_slider:
```

Comment:
Hi, why did you remove the animation speed slider example?

Reply:
Because I have merged the in to the white_value option of the light. This makes for a cleaner HA ui

Reply:
credit goes to thehookup on youtube, he was where i got the idea from.